libraries/node-core-library/src/RealNodeModulePath.ts

```diff
@@ -146,7 +150,17 @@ export class RealNodeModulePathResolver {
     // of an lstat call.
     const stat: nodeFs.Stats | undefined = this._fs.lstatSync(link);
     if (stat.isSymbolicLink()) {
-      return this._fs.readlinkSync(link, 'utf8');
+      const result: string = this._normalizeTrailingSlash(this._fs.readlinkSync(link, 'utf8'));
+      return result;
+    }
+  }
+
+  private _normalizeTrailingSlash(input: string): string {
+    const { _path: path } = this;
+    if (input.endsWith(path.sep)) {
```

Comment:
What if the provided path ends with a forward slash on Windows?

Reply:
It already underwent path normalization, unless you expressly put a `/` in a link target.

Reply:
I guess I could call `path.normalize` on the link target first if we want to handle that scenario expressly.